composer.json

```diff
@@ -64,7 +64,7 @@
         "ecs:test": "ecs check src --ansi --config vendor/symplify/easy-coding-standard/config/clean-code.yml",
         "phpstan:test": "phpstan analyse --ansi",
         "phpunit:test": "phpunit --colors=always",
-        "insights": "bin/phpinsights analyse --ansi -v --no-interaction --min-quality=85.7 --min-architecture=77.3 --min-style=89.1",
+        "insights": "bin/phpinsights analyse --ansi -v --no-interaction --min-quality=85.7 --min-architecture=77.3 --min-style=89.1 --disable-security-check",
```

Comment:
Why are you removing the security checks for the package? I get why you added support for removing it, but I think we should have it enabled on this repo.

Reply:
@olivernybroe See [travis build](https://travis-ci.org/nunomaduro/phpinsights/builds/534682598) 
When we launch it with prefer-lowest flag, the build fail. 

But maybe we should update the min-dependencies versions ? :thinking:

Reply:
@Jibbarth You should `--disable-security-check` only for min-dependencies.

Reply:
Okay, I have to search how to do that with travis. Maybe we have to launch several task instead just `composer run test`..

Reply:
Maybe @szepeviktor can help here.

Reply:
The only solution I see is to move it out of composer.json and put it in travis.yml's matrix.